index.html

```diff
@@ -0,0 +1,1753 @@
+              </path>
+            </g>
+          </svg>
+        </a>
+        <div class="col p-0">
+          <div class="row justify-content-end align-content-center text-uppercase font-weight-bold pr-0">
+            <a class="text-decoration-none text-white px-2 pt-2" href="#">Sign in</a>
+            <a class="text-decoration-none px-2 pt-2" style="background: white; color:#f72210; " href="#">Subscribe <i
```

Comment:
**[REQUIRED]** Remove inline styles. Add all your styles to the external CSS file.